### ISSUE TITLE
Calculate Accuracy By two Classifiers

### ISSUE THREAD

**Issue by GhostQ99** [NUST-Machine-Intelligence-Laboratory/Jo-SRC#2]
Hi, great inspired by your work. But I am curious about how to calculate the accuracy listed in the paper. Given two networks, is the result calculated by the mean accuracy of two classifiers or use the mean mixed logits of two networks as the final representations then calculate accuracy?
